### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by Marioalexsan**
Truly the commit of all time.

### CODE REVIEW

File: examples/cocoa/CocoaAppDelegate.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ////////////////////////////////////////////////////////////
 //
 // SFML - Simple and Fast Multimedia Library
-// Copyright (C) 2007-2023 Marco Antognini (antognini.marco@gmail.com),
+// Copyright (C) 2007-2024 Marco Antognini (antognini.marco@gmail.com),
```
Comment (ChrisThrasher) [inline examples/cocoa/CocoaAppDelegate.h:4]: I'm going to need you to double check it's 2024 by counting all years since 0

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Comment by ChrisThrasher** [comment]
Please update a file added in https://github.com/SFML/SFML/pull/1807

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]

**Review by ChrisThrasher — APPROVED** [review]

**Pull request merged**